spec/features/gis_accessioning_spec.rb

```diff
@@ -39,8 +40,8 @@
 
     # Go to kurma server and copy test content to the druid folder so it can be accessioned
     # Should this test data be deleted from the server,
-    # a zipped copy is available at spec/fixtures/gis_integration_test_data.zip
-    test_data_source_folder = File.join(Settings.gis.robots_content_root, 'integration_test_data')
+    # a zipped copy is available at spec/fixtures/gis_integration_test_data_vector.zip
+    test_data_source_folder = File.join(Settings.gis.robots_content_root, 'integration_test_data_vector')
     test_data_destination_folder = File.join(DruidTools::Druid.new(druid, Settings.gis.robots_content_root).path, 'content')
```

Comment:
new folder name for this existing test data to distinguish from raster based test data